docs/sources/send-data/k8s-monitoring-helm/_index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Kubernetes Monitoring Helm
+menuTitle: Kubernetes Monitoring Helm
+weight: 300
+description: Learn how to collect and store logs from your Kubernetes cluster using Loki.
+killercoda:
+  title: Kubernetes Monitoring Helm
+  description: Learn how to collect and store logs from your Kubernetes cluster using Loki.
+  backend:
+    imageid: kubernetes-kubeadm-2nodes
+---
+
+<!-- INTERACTIVE page intro.md START -->
+
+# Kubernetes Monitoring Helm
+
+One of the primary use cases for Loki is to collect and store logs from your [Kubernetes cluster](https://kubernetes.io/docs/concepts/overview/). These logs fall into three categories:
+
+1. [**Pod logs**](https://kubernetes.io/docs/concepts/cluster-administration/logging/#basic-logging-in-kubernetes): Logs generated by containers otherwise known as logs running in your cluster.
```

Comment:
Should the second part of the sentence be "Pods running in your cluster"?   You refer to "pod logs" down at line 25....

```suggestion
1. [**Pod logs**](https://kubernetes.io/docs/concepts/cluster-administration/logging/#basic-logging-in-kubernetes): Logs generated by containers, otherwise known as logs running in your cluster.
```